src/switch_ivr_bridge.c

```diff
@@ -65,6 +67,42 @@ static void text_bridge_thread(switch_core_session_t *session, void *obj)
 
 	vh->up = 1;
 
+	if (!vh->session_a) {
```

Comment:
i missed this in my previous reviews - i think this is still possibly unsafe, since a pointer to a session is being passed to a different thread (and without a lock being held), there is no guarantee that the pointer will be valid after a context switch. me thinks it is safer to only use `switch_core_session_locate` in order to get a pointer to a session. @dev-ryanc what do you think? thanks